### PR TITLE
fix back button on ListSettingsView

### PR DESCRIPTION
**Pull request by xdeelord24**
### What does it do?

image

### Why is it needed?

The back button is not working, and I need it to facilitate easy back history.

### How to test it?

The back button does not work when configuring the view of any collection or single type on ListSettingsView.

### Related issue(s)/PR(s)

There is no related issue

### CODE REVIEW

**Comment by joshuaellis** [comment]
Can you elaborate on why the back-button doesn't currently work for you?

**Comment by xdeelord24** [comment]
> Can you elaborate on why the back-button doesn't currently work for you?

When I create new project on strapi@latest as V4, I was editing some collections and I was gonna change the view of my viewList data using "Configure the view" , after editing the fields I want to click the back button and suddenly it is not working, I also tried the previous version of V4 and it is all not working. You can try the latest V4.
This is the button that I'm fixing. 
image

It works when I change the code into this PR

**Review by joshuaellis — CHANGES_REQUESTED** [review]
I think the solution is actually to change L187 from `to` to `pathname` as per the types of `Link` from react-router-dom. If we can do that change, which is a smaller surface area to change, we can accept this :)

File: packages/core/admin/admin/src/content-manager/pages/ListSettingsView/index.jsx [hunk 1]
```diff
@@ -183,23 +171,16 @@ export const ListSettingsView = ({ layout, slug }) => {
             navigationAction={
               <Link
                 startIcon={<ArrowLeft />}
-                to={{
-                  to: `/content-manager/${kind}/${uid}`,
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/content-manager/pages/ListSettingsView/index.jsx:187]: Sorry there was some confusion, if you dont make the changes you've made and just change this property to `pathname` it should work as you expect it too?

Comment (xdeelord24) [reply packages/core/admin/admin/src/content-manager/pages/ListSettingsView/index.jsx:187]: it works for me

Comment (xdeelord24) [reply packages/core/admin/admin/src/content-manager/pages/ListSettingsView/index.jsx:187]: it works, I'll PR it

**Review by joshuaellis — APPROVED** [review]

**Comment by joshuaellis** [comment]
Looks like prettier is failing. You need to format that file 👍🏻

**Pull request merged**